apiserver/facades/client/client/filtering.go

```diff
@@ -99,7 +99,7 @@ func BuildPredicateFor(patterns []string) Predicate {
 	return func(i interface{}) (bool, error) {
 		switch i.(type) {
 		default:
-			panic(errors.Errorf("Programming error. We should only ever pass in machines, applications, or units. Received %T.", i))
+			panic(errors.Errorf("Programming error. We should only ever pass in machines, applications or units. Received %T.", i))
```

Comment:
say it isn't so. I'm an Oxford Comma man, myself. https://en.oxforddictionaries.com/explore/what-is-the-oxford-comma

(apparently legally you're required to use them in some lists if it aids clarity). I realize certain official styles prefer to not have them.
I don't think we have a style guide in Juju around one or the other.